src/renderer/entities/asset/ui/AssetCard/AssetCard.tsx

```diff
@@ -71,7 +72,7 @@ export const AssetCard = ({ chainId, asset, balance, canMakeActions }: Props) =>
             </div>
           )}
         </div>
-        {canMakeActions && (
+        {!walletUtils.isWatchOnly(activeWallet) && (
```

Comment:
You can create variable in model, like `$activeWallet` or `$activeAccounts` but they have different styles and better to ask @tuul-wq which one is correct

Reply:
We can use local model variable when effector model will be implemented for `Assets` page
For now it's absolutely ok to use domain function, especially without props drill 👍